python/ray/data/_internal/logical/operators/one_to_one_operator.py

```diff
@@ -0,0 +1,52 @@
+from typing import Optional
+
+from ray.data._internal.logical.interfaces import LogicalOperator
+
+
+class AbstractOneToOne(LogicalOperator):
+    """Abstract class for one-to-one logical operators should
+    be converted to their corresponding physical operators.
```

Comment:
```suggestion
    """Abstract class for one-to-one logical operators.
```
The second part of the sentence doesn't seem useful here.